REUSE.toml

```diff
@@ -283,3 +283,9 @@ path = ["ui-libraries/material/src/ui/icons/**.svg", "ui-libraries/material/exam
 precedence = "aggregate"
 SPDX-FileCopyrightText = "Material Icons <https://github.com/material-icons/material-icons/blob/master/LICENSE>"
 SPDX-License-Identifier = "Apache-2.0"
+
+[[annotations]]
+path = ["docs/astro/src/assets/android/**.png"]
+precedence = "aggregate"
+SPDX-FileCopyrightText = "Copyright © SixtyFPS GmbH <info@slint.dev>"
+SPDX-License-Identifier = "Unlicense"
```

Comment:
I'm curious, is there a particular reason to use Unlicensed instead of `CC-BY-ND-4.0` that we've used for other assets that "come from us" in the docs?

Reply:
They're just screenshots of an application, not something I've drawn myself.